### PR TITLE
net: wifi_utils: Fix wifi_utils_parse_scan_ssids doc

### PR DESCRIPTION
**Pull request by ceolin**
Function recently changed its behavior, just changing the documentation to reflect that.

### CODE REVIEW

**Comment by kartben** [comment]
If I understand the new behaviour then it looks like the function should actually be renamed. It's not "parsing", and it's not ssid*s*. Probably something something append ssid?

File: include/zephyr/net/wifi_utils.h [hunk 1]
```diff
@@ -57,12 +57,12 @@ int wifi_utils_parse_scan_bands(char *scan_bands_str, uint8_t *band_map);
 
 
 /**
- * @brief Convert a string containing a list of SSIDs to an array of SSID strings.
+ * @brief Include a string containing a SSID to an array of SSID strings.
```
Comment (kartben) [inline include/zephyr/net/wifi_utils.h:60]: Use "add" or "append"
"a SSID" ---> "an SSID"

File: include/zephyr/net/wifi_utils.h [hunk 2]
```diff
@@ -57,12 +57,12 @@ int wifi_utils_parse_scan_bands(char *scan_bands_str, uint8_t *band_map);
...
  *
- * @details The function will parse a string which specifies Wi-Fi SSIDs
- * as a comma separated string and convert it to an array.
+ * @details The function will add a string which specifies Wi-Fi SSID
+ * to an array of SSIDs.
```
Comment (kartben) [inline include/zephyr/net/wifi_utils.h:63]: This is not really adding anything the brief didn't already mention so maybe just drop

File: include/zephyr/net/wifi_utils.h [hunk 3]
```diff
@@ -57,12 +57,12 @@ int wifi_utils_parse_scan_bands(char *scan_bands_str, uint8_t *band_map);
...
  *
- * @param scan_ssids_str List of SSIDs expressed as a comma separated list.
+ * @param scan_ssids_str string to be included in the list of scanned SSID.
```
Comment (kartben) [inline include/zephyr/net/wifi_utils.h:65]: Ditto -- use added or appended instead of included

**Review by kartben — DISMISSED** [review]

**Comment by ceolin** [comment]
@kartben I will leave the renaming for the maintainer, the documentation shouldn't change if it gets renamed.
It just happened that I saw this code two days ago and  a commit  yesterday was changing it.

**Comment by kartben** [comment]
> @kartben I will leave the renaming for the maintainer, the documentation shouldn't change if it gets renamed. It just happened that I saw this code two days ago and a commit yesterday was changing it.

yes that's fair. 
also flagging @JordanYates who I think is the one who introduced the change. Can you please check if the naming of `wifi_utils_parse_scan_ssids` and its params still makes sense? It looks very confusing to me?

**Review by jukkar — APPROVED** [review]
IMHO the function should/could be renamed, it is no longer do any parsing but just storing string to an array.

**Review by rlubos — APPROVED** [review]

**Pull request merged**